src/includes/test-verify/javascript.mdx

```diff
@@ -0,0 +1,7 @@
+```JavaScript {tabTitle: ESM}
+import * as Sentry from "@sentry/browser";
+
+Sentry.init({ dsn: "___PUBLIC_DSN___" });
+
+myUndefinedFunction(); // To test your setup
```

Comment:
Should we just reduce it to this?
Since the snippet above already contains everything else.

```suggestion
myUndefinedFunction(); // To test your setup
```

Reply:
Yes, I pulled that content from old docs, so wasn't quite sure whether we should have the full set of lines or the simple error line. Thanks for checking.